code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -341,7 +341,19 @@ GLOBAL_LIST_INIT(cardboard_recipes, list (
 	new /datum/stack_recipe("folder", /obj/item/folder),
 	new /datum/stack_recipe("cardboard tube", /obj/item/c_tube),
 	new /datum/stack_recipe("cardboard box", /obj/structure/closet/cardboard, 4),
-	new /datum/stack_recipe("cardboard cutout", /obj/item/cardboard_cutout, 5)
+	new /datum/stack_recipe("cardboard cutout", /obj/item/cardboard_cutout, 5),
+	new /datum/stack_recipe("black pawn", /obj/item/chesspiece/bpawn, 1),
```

Comment:
```suggestion
	null,
	new /datum/stack_recipe("black pawn", /obj/item/chesspiece/bpawn, 1),
```

Add a `null` to separate the above stuff with the chess thingies

Reply:
Done~